src/portlet/ClassNames.tsx

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright (c) 2018 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+export class Styles {
+    public static ICON_STYLE = {transition: "all .3s ease-in 0s", color: "#007DB8"};
+    public static MAIN_DIV: any = {
```

Comment:
No need to add explicit 'any' type here.

Reply:
yes, it is required at it is throwing error for some of the properties